<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MuPlayStatusBarUI</class>
 <widget class="QWidget" name="MuPlayStatusBarUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>70</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>60</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <property name="spacing">
    <number>30</number>
   </property>
   <property name="leftMargin">
    <number>35</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>30</number>
     </property>
     <item alignment="Qt::AlignVCenter">
      <widget class="QPushButton" name="preSongBtn">
       <property name="minimumSize">
        <size>
         <width>38</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>38</width>
         <height>38</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#bbbb {
	background-color: rgb(73, 112, 255);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignVCenter">
      <widget class="QPushButton" name="playBtn">
       <property name="minimumSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignVCenter">
      <widget class="QPushButton" name="nextSongBtn">
       <property name="minimumSize">
        <size>
         <width>38</width>
         <height>38</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>38</width>
         <height>38</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="curTimeLb">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MuBaseSlider" name="playSlider">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>166666</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sumTimeLb">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="voiceBtn">
       <property name="minimumSize">
        <size>
         <width>34</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>34</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MuVoiceSlider" name="voiceSlider">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QPushButton" name="playOrderBtn">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lyricsBtn">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>lyric</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openListBtn">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MuVoiceSlider</class>
   <extends>QSlider</extends>
   <header>muplaystatusbarui.h</header>
  </customwidget>
  <customwidget>
   <class>MuBaseSlider</class>
   <extends>QSlider</extends>
   <header>muplaystatusbarui.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
